using-mobile-extensions/adobe-target/README.md

```diff
@@ -422,25 +451,36 @@ ACPCore.collectLaunchInfo(["adb_deeplink" : "com.adobe.targetpreview://app.adobe
 {% endtab %}
 {% endtabs %}
 
+## Offer Prefetch
+
+The SDK can minimize the number of times it reaches out to Target servers to fetch offers by caching server responses. With a successful prefetch call for mbox locations, offer content is retrieved and cached in the SDK. This content is retrieved from the cache for all future [retrieveLocationContent](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#retrieveLocationContent) API calls for the specified mbox names. This prefetch process reduces the offer load time, and network calls that were made to the Target servers, and allows Target to be notified which mbox was visited by the mobile app user.
```

Comment:
- In the 3rd sentence, please remove the comma between "time" and "and".
- Please replace "servers, and allows Target..." with "servers, and **the process** allows Target..."